sky/backends/cloud_vm_ray_backend.py

```diff
@@ -2127,6 +2132,7 @@ def __init__(
             stable_internal_external_ips: Optional[List[Tuple[str,
                                                               str]]] = None,
             stable_ssh_ports: Optional[List[int]] = None,
+            cluster_info: Optional[provision_common.ClusterInfo] = None,
```

Comment:
```suggestion
            cached_cluster_info: Optional[provision_common.ClusterInfo] = None,
```
to align w/ variable name

Reply:
The argument that passed in is not cached, but only after it is assigned to `self.cached_cluster_info` we are caching the object, so probably we should not change the name here.